### PR TITLE
Add ci workflow columns in readme

### PR DESCRIPTION
**Pull request by sip-21**

### CODE REVIEW

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -91,7 +91,7 @@ jobs:
         export CLN_PATH=${{ github.workspace }}/lightning
         pip3 install --upgrade pip
         pip3 install --user -U virtualenv pip > /dev/null
-        python3 .ci/test.py
+        python3 .ci/test.py nightly true
```
Comment (chrisguida) [inline .github/workflows/nightly.yml]: Can we change `true` to something more descriptive?

Perhaps `badges-true`

Comment (sip-21) [reply .github/workflows/nightly.yml]: I turned it into a boolean flag `--update-badges`

Comment (chrisguida) [reply .github/workflows/nightly.yml]: Perfect!

**Comment by chrisguida** [comment]
Please add description and squash

**Comment by sip-21** [comment]
> Please add description and squash

@chrisguida I guess you saw my PR here: https://github.com/lightningd/plugins/pull/509
Will close this one.

**Pull request closed without merging**